### ISSUE TITLE
Release confusion

### ISSUE THREAD

**Issue by blimmer** [embersherpa/ember-cli-app-version#20]
I'm really confused why the branch is tagged as 0.4 and the package.json is still marked as 0.3.4. Should the new version be bumped and published?

**Comment by taras** [comment]
Yeah, I forgot to push master. Sorry :)

**Comment by taras** [comment]
Should be good now. Please check

**Comment by blimmer** [comment]
Can you also please publish 0.3.4? I'm on Ember 1.11 and can't use instance initializers yet. It's defined [here](https://github.com/embersherpa/ember-cli-app-version/commit/60fa1168f740f4e1f744a74f9a78c6546e90bd39) and it's not published according to npm:
```
  time:
   { modified: '2015-06-27T02:05:10.471Z',
     created: '2014-11-04T20:07:22.745Z',
     '0.1.0': '2014-11-04T20:07:22.745Z',
     '0.1.1': '2014-11-11T20:53:06.495Z',
     '0.2.0': '2014-11-18T21:45:10.566Z',
     '0.2.1': '2014-12-04T22:29:10.523Z',
     '0.2.2': '2015-01-08T22:54:30.985Z',
     '0.2.3': '2015-01-25T15:56:31.879Z',
     '0.3.0': '2015-01-26T21:46:28.913Z',
     '0.3.1': '2015-01-28T15:02:35.862Z',
     '0.3.2': '2015-03-07T16:50:18.502Z',
     '0.3.3': '2015-03-10T14:10:50.681Z',
     '0.4.0': '2015-06-27T02:05:10.471Z' },
```

**Comment by taras** [comment]
Ok, I just pushed 0.3.5. You can use that :)

**Comment by truenorth** [comment]
Just got snagged by this as well. Brand new addons from the default ember-cli blueprint are no longer ember 1.10 or 1.11 compatible due to this dependency. This should probably not be the default, since 1.11 is not all that old yet

**Comment by taras** [comment]
All of this should be resolved as of 0.5.0 which supports pre Instance Initializer versions of Ember and post. Please, give it a shot and let me know.

**Comment by mike-north** [comment]
awesome, thanks taras!
